x/accounts/vesting/continuous_vesting_account.go

```diff
@@ -0,0 +1,191 @@
+package vesting
+
+import (
+	"context"
+	"time"
+
+	"cosmossdk.io/collections"
+	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	"cosmossdk.io/x/accounts/accountstd"
+	vestingtypes "cosmossdk.io/x/accounts/vesting/types/v1"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Compile-time type assertions
+var (
+	_ accountstd.Interface = (*ContinuousVestingAccount)(nil)
+)
+
+// Continuos Vesting Account
+
+// NewContinuousVestingAccount creates a new ContinuousVestingAccount object.
+func NewContinuousVestingAccount(d accountstd.Dependencies) (*ContinuousVestingAccount, error) {
+	baseVestingAccount, err := NewBaseVestingAccount(d)
+
+	continuousVestingAccount := ContinuousVestingAccount{
+		BaseVestingAccount: baseVestingAccount,
```

Comment:
There is not a need for a base vesting account. These are all different vesting accounts.

Reply:
hmmm i think the base vesting account is just to define all the similar field and methods of other different vesting accounts